### ISSUE TITLE
breaking changes in underlying node-rdkafka library

### ISSUE THREAD

**Issue by hauntingEcho** [hjespers/node-red-contrib-rdkafka#3]
due to the changes in [this commit](https://github.com/Blizzard/node-rdkafka/commit/dba23dc9f00bc310cc468650fe312115c7486d81#diff-69b3da3df139e0ac86797a46df0ef970) of node-rdkafka, node-red-contrib-rdkafka is no longer using the underlying library correctly.

This can currently be addressed by installing node-rdkafka@0.5.5 over the default-installed dependency (@0.6.0, released Nov 18)

**Comment by hjespers** [comment]
for now I updated the dependancies to specific node-rdkafka 0.5.5. I will go back and update the producer for the new API format

**Comment by hjespers** [comment]
the issue must have been a 0.6.0 bug because I use the streams API and it didn't change. I tested against node-rdkafka 0.6.2 and its working for me. Since Blizzard fixed the issue of hanging on disconnect I can now properly disconnect all consumers upon exit. This is a nice enhancement since before consumers had to time out of consumer groups rather than leaving gracefully.

Test out the lastest version I published to npm (0.0.3) and reopen if you see any compatibility problems.
